git/08-share.md

```diff
@@ -178,6 +181,7 @@ What are some of the benifits of using version control, Git and GitLab?
 
 {% endchallenge %}
 
+
```

Comment:
You could remove the diff in this file; the newlines probably don't change anything.

Reply:
The current html website puts consecutive md lines with no spacing on the line. This means that the boxes don't render properly. 

The result looks like this 
![image](https://user-images.githubusercontent.com/33899169/97305069-e9975900-185c-11eb-88db-aa6a22de25ac.png)

I was hoping these extra spacing would solve the issue.

Reply:
Ah yes. We do need the first newline you've added, before `{% solution "Solution" %}` but the rest are not needed.

Reply:
Fair, I will remove those.